interface/modules/custom_modules/oe-module-faxsms/openemr.bootstrap.php

```diff
@@ -73,6 +73,16 @@ function oe_module_faxsms_add_menu_item(MenuEvent $event): MenuEvent
     $menuItem->children = [];
     $menuItem->acl_req = ["patients", "docs"];
     $menuItem->global_req = ["oefax_enable_sms"];
+    // Our Email menu
+    $menuItemEmail = new stdClass();
+    $menuItemEmail->requirement = 0;
+    $menuItemEmail->target = 'email';
+    $menuItemEmail->menu_id = 'email';
+    $menuItemEmail->label = xlt("Email");
```

Comment:
Would you be more descriptive because one could assume they can email others through clinic email SMTP. For now, unless you want to add, we don't have that capability though I've had that capability requested more than once.

Eventually I want to have an option for which SMTP to use as I've started that process in email setup.